src/bmf.cpp

```diff
@@ -204,16 +206,17 @@ bool CxadbmfPlayer::xadplayer_load()
     }
     ptr += 32 * 15;
   }
-  
+
   // load streams
   if (bmf.version > BMF0_9B)
   {
     if (tune_size - ptr < 4)
       return false;
-    unsigned long sflags = (tune[ptr] << 24) | (tune[ptr+1] << 16) | (tune[ptr+2] << 8) | tune[ptr+3];
-    ptr+=4;
+    unsigned long sflags = (tune[ptr] << 24) | (tune[ptr+1] << 16)
+                         | (tune[ptr+2] << 8) | tune[ptr+3];
```

Comment:
Does this look neater if rather than shortening it for the sake of line length, you just put each `tune[]` thing on its own line?  Might make it easier to see the pattern?

Reply:
Actually this should not be open coded at all. This is a pattern that's used all over the place and many players have their own macros/functions dealing with it in addition to libbinio providing a solution for file data. All that should be consolidated into a common set of helpers used everywhere. But that's out of scope for this PR.

Reply:
Fair enough!